app/src/main/res/values-de/strings.xml

```diff
@@ -4,7 +4,9 @@
     <string name="app_name">Prepaid-Guthaben</string>
     <string name="ussd_failed">Abfragen des USSD-Codes fehlgeschlagen</string>
     <string name="unable_get_balance">Auslesen des Guthabens fehlgeschlagen: %s</string>
-    <string name="permissions_required">Telefon-Berechtigung benötigt</string>
+    <string name="permissions_required_phone">Telefon-Berechtigung benötigt</string>
+	<string name="permissions_required_notification">Berechtigung für Benachrichtigungen benötigt</string>
+    <string name="permissions_required_phone_and_notification">Berechtigung für Telefon und Benachrichtigungen benötigt</string>
```

Comment:
Can you revert all changes in `values-*/strings.xml`? These files are updated by Crowdin.

Reply:
done